apps/docs/src/components/ShowcaseCard.astro

```diff
@@ -0,0 +1,24 @@
+---
+const { title, href } = Astro.props;
+---
+
+<a
+  class="relative flex flex-col rounded bg-neutral-100 dark:bg-neutral-800 hover:bg-brand-primary-200 dark:hover:bg-brand-primary-600 transition-color duration-300"
+  role="button"
+  href={href}
+>
+  <div class="px-2 pt-2">
+    <figure
+      class="flex justify-center items-center h-9 sm:h-8 w-full bg-white rounded overflow-hidden"
+    >
+      <img src={`/${href}-showcase.png`} class="object-fit" />
+    </figure>
+  </div>
+  <div class="px-2 pb-2 text-sm flex flex-1 flex-col justify-end">
+    <h2
+      class="flex items-center gap-0.5 text-base leading-relaxed font-semibold"
+    >
+      {title || 'Sample Title'}
```

Comment:
why the `||`? I'd expect to always have title since it doesn't makes sense without it